tests/Client/UpdaterTest.php

```diff
@@ -63,27 +93,25 @@ protected function getSystemInTest() : Updater
      *
      * @param string $fixturesSet
      *   The fixtures set to use.
-     * @param array $expectedStartVersions
-     *   The expected start versions.
      * @param array $expectedUpdatedVersions
      *   The expected updated versions.
      *
      * @return void
      *
      * @dataProvider providerRefreshRepository
      */
-    public function testRefreshRepository(string $fixturesSet, array $expectedStartVersions, array $expectedUpdatedVersions) : void
+    public function testRefreshRepository(string $fixturesSet, array $expectedUpdatedVersions) : void
```

Comment:
I removed `$expectedStart` version because this doesn't vary with every test case but instead with the feature set used. So this can be determined from `$fixturesSet`.

Added `getFixtureClientStartVersions()` so now every test in the class can assert the starting point based on the feature set.